lib/html_to_portable_text/get_rnd_block_key.ts

```diff
@@ -0,0 +1,6 @@
+import crypto from "crypto"
+
+export const getRndBlockKey = (l = 16): string => {
```

Comment:
Would be easier to read if we just use random :)

Reply:
true